actions/approve_csr_handler.go

```diff
@@ -122,25 +125,23 @@ func (h *approveCSRHandler) getInitialNodeCSR(ctx context.Context, log logrus.Fi
 	var cert *csr.Certificate
 	var err error
 
-	logRetry := func(err error, _ time.Duration) {
-		log.Warnf("getting initial csr, will retry: %v", err)
-	}
-	b := backoff.WithMaxRetries(backoff.NewExponentialBackOff(), 3)
-	err = backoff.RetryNotify(func() error {
+	// TODO: Shouldn't we pass ctx here, too?
```

Comment:
I left this here to discuss - We use a context in the function above. Shouldn't we pass the context here as well? 
Original code did not do that.

Reply:
Thanks that's a good catch. I think it is a good idea if we pass it.